toolboxes/mri_core/mri_core_export.h

```diff
@@ -0,0 +1,16 @@
+/** \file       mri_core_export.h
```

Comment:
Still don't understand why this file is needed. You are not exporting anything, right?